apps/gallery/test/marionette/apps/fakeactivitycaller/manifest.webapp

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Fake Gallery Activity Caller",
+  "description": "Fake Gallery activity caller app",
+  "type": "certified",
+  "launch_path": "/index.html",
+  "developer": {
+    "name": "The Gaia Team",
+    "url": "https://github.com/mozilla-b2g/gaia"
+  },
+  "activities": {
+    "share": {
+      "href": "/index.html#activity-share",
+      "filters": {
+        "type": ["image/*"],
+        "number": {
+          "max": 5
```

Comment:
Since you're not actually testing this number thing, I'd just leave it out of the manifest.